2022-05-16 06:40:16,120	INFO head.py:41 -- Dashboard head grpc address: 192.168.50.44:54867
2022-05-16 06:40:16,120	WARNING dashboard.py:98 -- [Errno 2] Dashboard build directory not found. If installing from source, please follow the additional steps required to build the dashboard(cd python/ray/new_dashboard/client && npm install && npm ci && npm run build): 'c:\\users\\janguy\\anaconda3\\envs\\dlresearch\\lib\\site-packages\\ray\\new_dashboard\\client\\build'
2022-05-16 06:40:16,125	INFO head.py:82 -- Connect to GCS at b'192.168.50.44:54855'
2022-05-16 06:40:16,125	INFO utils.py:202 -- Get all modules by type: DashboardHeadModule
2022-05-16 06:40:16,595	ERROR dashboard.py:237 -- The dashboard on node JAMESN failed with the following error:
Traceback (most recent call last):
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\dashboard.py", line 222, in <module>
    loop.run_until_complete(dashboard.run())
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\dashboard.py", line 118, in run
    await self.dashboard_head.run()
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\head.py", line 106, in run
    modules = self._load_modules()
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\head.py", line 47, in _load_modules
    head_cls_list = dashboard_utils.get_all_modules(
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\utils.py", line 208, in get_all_modules
    importlib.import_module(name)
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\modules\log\test_log.py", line 13, in <module>
    from ray.new_dashboard.tests.conftest import *  # noqa
  File "c:\users\janguy\anaconda3\envs\dlresearch\lib\site-packages\ray\new_dashboard\tests\conftest.py", line 3, in <module>
    from ray.tests.conftest import *  # noqa
ModuleNotFoundError: No module named 'ray.tests'

